optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/realworld/conferencescheduling/ConferenceSchedulingBenchmark.java

```diff
@@ -28,7 +28,7 @@
     private static final String CONFERENCE_SCHEDULING_DROOLS_SCORE_RULES_FILE =
             "org/jboss/qa/brms/performance/examples/conferencescheduling/solver/conferenceSchedulingScoreRules.drl";
 
-    @Param({"TALKS_36_TIMESLOTS_12_ROOMS_5", "TALKS_108_TIMESLOTS_18_ROOMS_10", "TALKS_216_TIMESLOTS_18_ROOMS_20"})
+    @Param({"DAY_2_ROOM_5", "DAY_3_ROOM_10", "DAY_3_ROOM_20"})
```

Comment:
I would rather keep the original names - it gives you good idea about the space of the problem being solved. "DAY_2_ROOM_5" tells you nothing.